docs/runbooks/CDIStorageProfilesIncomplete.md

```diff
@@ -60,3 +60,11 @@ status:
   provisioner: kubernetes.io/no-provisioner
   storageClass: local
 ```
+
```

Comment:
I don't think we need the "submit an issue" section. The user should be able to fix the problem.